model/trigger/task.go

```diff
@@ -213,14 +213,28 @@ func (t *taskTriggers) makeData(sub *event.Subscription, pastTenseOverride strin
 }
 
 func (t *taskTriggers) generate(sub *event.Subscription, pastTenseOverride string) (*notification.Notification, error) {
-	data, err := t.makeData(sub, pastTenseOverride)
-	if err != nil {
-		return nil, errors.Wrap(err, "failed to collect task data")
-	}
+	var payload interface{}
+	if sub.Subscriber.Type == event.JIRAIssueSubscriberType {
```

Comment:
I think we should probably push this as is, but should open a ticket for the future to refactor things so that triggers don't need to know about how to produce their output forms, or can convert maps (string:interface) into their needed forms (maybe even pushing this down to grip.